src/client/Microsoft.Identity.Client.Broker/WamAdapters.cs

```diff
@@ -281,5 +288,28 @@ private static string GetExpectedRedirectUri(string clientId)
         {
             return $"ms-appx-web://microsoft.aad.brokerplugin/{clientId}";
         }
+
+        /// <summary>
+        /// Validate common auth params
+        /// </summary>
+        /// <param name="authenticationRequestParameters"></param>
+        /// <param name="logger"></param>
+        /// <exception cref="MsalClientException"></exception>
+        public static void ValidateAuthParams(
+            AuthenticationRequestParameters authenticationRequestParameters,
+            ILoggerAdapter logger)
+        {
+            //MSAL Runtime throws an ApiContractViolation Exception with Tag: 0x2039c1cb (InvalidArg)
+            //When no scopes are passed, this will check if user is passing scopes
+            if (!authenticationRequestParameters.HasScopes)
```

Comment:
The app developer can always set one of the `profile`, `openid`, `offline_access` and that bug will most likely repro. So you probably want to make that the scopes are not a subset of the OIDC_scopes.

Reply:
the bug does not repro when any one of the those scopes are passed (`profile, openid, offline_access`), but it is good to check in case something changes in the cpp layer.